sanic/mixins/routes.py

```diff
@@ -665,7 +665,10 @@ async def _static_request_handler(
                 modified_since = strftime(
                     "%a, %d %b %Y %H:%M:%S GMT", gmtime(stats.st_mtime)
                 )
-                if request.headers.get("If-Modified-Since") == modified_since:
+                if (
+                    request.headers.getone("If-Modified-Since", None)
```

Comment:
While you are at it, maybe we should be consistent. Sometimes when we access a header we ask for it in title-case. Sometimes lower case. I think it would be nice to just change all these to lower case strings.